api/controllers/SdtdServer/countryBan/disable-country-whitelist.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+
+  friendlyName: 'Disable country whitelist',
+
+  description: 'Set the country banlist to act as a blacklist',
+
+  inputs: {
+    serverId: {
+      type: 'number',
+      required: true
+    }
+  },
+
+  exits: {
+    success: {
+    },
+  },
+
+
+  fn: async function (inputs, exits) {
+    let config = await SdtdConfig.findOne({ server: inputs.serverId });
+    let countryBanListMode = config.countryBanListMode;
+
+    countryBanListMode.countryBanListMode = false;
```

Comment:
This seems weird, why is this not

`countryBanListMode = false`

?

Reply:
It should be yeah, I don't know why I did that.